common/src/main/java/com/graphicsfuzz/common/util/FragmentShaderJobToVertexShaderJob.java

```diff
@@ -77,6 +81,27 @@ public static ShaderJob convertShaderJob(ShaderJob sj) throws Exception {
     TranslationUnit vertexShader = sj.getFragmentShader().get();
     vertexShader.setShaderKind(ShaderKind.VERTEX);
 
+    // Analyse the shader for used features (can't use anonymous class due to member access)
+    class Analysis extends StandardVisitor {
+      public boolean usesFragCoord = false;
+      public boolean usesFragDepth = false;
```

Comment:
Please make private and add getUsesFragCoord() and getUsesFragDepth() methods.  A bit annoying I know, but I try to enforce that fields are private throughout the project.

Reply:
done